src/Stratis.Bitcoin/Stratis.Bitcoin.csproj

```diff
@@ -20,6 +20,12 @@
     <Authors>Stratis Group Ltd.</Authors>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
This weird folder has been in the solution for a long time, I can remove this from the PR if necessary but I'm not sure what the folder is actually for, if anything.